### PR TITLE
Fix nextflow using local executor, not SGE

### PR DESCRIPTION
**Pull request by Valiec**
I added a nextflow.config file setting the executor to SGE, so that the pipeline doesn't just all run on one cluster node, but actually qsubs the tasks.

I'm test-running this config on my pangolin test (not loading to the DB yet) on another branch, but it's already qsubbing the jobs, so it should work.

### CODE REVIEW

**Comment by Valiec** [comment]
Update: It crashed on not finding the python 'regex' module. Still looking into if that's due to the different environment (I'm running it from hpc-l1 with a nextflow install from there currently, with the DB load steps turned off), or because of the SGE stuff. Don't merge this yet.

**Comment by Valiec** [comment]
I think this will work. I haven't gotten my test all the way through, because the align step is currently stuck waiting in the cluster queue, but it's worked on previous tests.

**Comment by Valiec** [comment]
The test with pangolin and no db load worked, so I think the nextflow changes here (which are pretty much the same) should be OK.

**Comment by bwlang** [comment]
i think penv = 'smp' can go into the config since it's the same everywhere .  if that's right we can remove penv from the individual processes

File: lib/nextflow.config [hunk 1]
```diff
@@ -1,2 +1,3 @@
 process.executor = 'sge'
+process.executor.penv = 'smp'
```
Comment (Valiec) [inline lib/nextflow.config:2]: My test (on DSL2 though) just says "No such variable: penv" unfortunately.

Comment (Valiec) [reply lib/nextflow.config:2]: Same error on start for DSL1 with a simple test workflow,

Comment (bwlang) [reply lib/nextflow.config:2]: ah. it seems to be at the same level 
maybe process.pend = 'smp'

Comment (Valiec) [reply lib/nextflow.config:2]: I'll try that this morning and see.

Comment (Valiec) [reply lib/nextflow.config:2]: Seems to work!

**Review by bwlang — APPROVED** [review]

**Pull request merged**